course-offline/assets/course-offline.ts

```diff
@@ -18,6 +17,7 @@ import "nanogallery2/src/jquery.nanogallery2.core.js"
 import "nanogallery2/src/css/nanogallery2.css"
 import "videojs-youtube"
 import videojs from "video.js"
+import _ from "lodash"
```

Comment:
@ibrahimjaved12 My apologies for suggesting a lodash approach. I had been under the impression we were already using lodash in the course bundle, but I guess we aren't. Given that, I think it's better to not use `_.once` and just avoid the double initialization with a simple flag.